packages/react-core/src/components/TimePicker/TimePicker.tsx

```diff
@@ -324,15 +354,15 @@ export class TimePicker extends React.Component<TimePickerProps, TimePickerState
     });
   };
 
-  onSelect = (selection: string) => {
+  onSelect = (e: any) => {
```

Comment:
I think this type needs to be more specific.

Reply:
I was wrong! This is the event passed to MenuItem, which has an `onClick` that uses `e: any`. This needs to stay the way it is! :)